dokka-subprojects/core/src/main/kotlin/org/jetbrains/dokka/links/DRI.kt

```diff
@@ -146,6 +146,10 @@ public data class Nullable(val wrapped: TypeReference) : TypeReference() {
     override fun toString(): String = "$wrapped?"
 }
 
+public data class Vararg(val elementType: TypeReference) : TypeReference() {
```

Comment:
Are the links in KDoc `[java.util.Arrays.asList]` and Javadoc `{@link java.util.Arrays#asList}` consistent?

Reply:
And DRI?
I had supposed the PSI parser to DRI should be changed as well.  https://github.com/Kotlin/dokka/blob/60236e9c6497ef00aa060df1c77d41e0d74b18f9/dokka-subprojects/analysis-java-psi/src/main/kotlin/org/jetbrains/dokka/analysis/java/util/PsiUtil.kt#L20

Reply:
Indeed, `JavaClassReference` can store arbitrary text. Nothing has broken so far.

> But I'm not sure that it should be the part of this PR

I think it’s better to leave it as is for now.

Reply:
I don't think that it is covered by #3502, but I will check and create a new issue if necessary